test/integration/model/find.test.js

```diff
@@ -760,104 +759,71 @@ describe(Support.getTestDialectTeaser('Model'), function() {
-                , tags = results.tags;
-
-              async.parallel([
-                function(callback) {
-                  products[0].setTags([tags[0], tags[1]]).done(callback);
-                },
-                function(callback) {
-                  products[1].addTag(tags[0]).done(callback);
-                },
-                function(callback) {
-                  products[2].addTag(tags[1]).done(callback);
-                },
-                function(callback) {
-                  products[3].setTags([tags[1]]).done(callback);
-                },
-                function(callback) {
-                  products[4].setTags([tags[2]]).done(callback);
-                }
-              ], function(err) {
-                expect(err).not.to.exist;
-
-                async.parallel([
-                  function(callback) {
-                    self.Tag.find({
-                      where: {
-                        id: tags[0].id
-                      },
-                      include: [
-                        {model: self.Product, as: 'products'}
-                      ]
-                    }).done(function(err, tag) {
-                      expect(tag).to.exist;
-                      expect(tag.products.length).to.equal(2);
-                      callback();
-                    });
-                  },
-                  function(callback) {
-                    tags[1].getProducts().done(function(err, products) {
-                      expect(products.length).to.equal(3);
-                      callback();
-                    });
-                  },
-                  function(callback) {
-                    self.Product.find({
-                      where: {
-                        id: products[0].id
-                      },
-                      include: [
-                        {model: self.Tag, as: 'tags'}
-                      ]
-                    }).done(function(err, product) {
-                      expect(product).to.exist;
-                      expect(product.tags.length).to.equal(2);
-                      callback();
-                    });
-                  },
-                  function(callback) {
-                    products[1].getTags().done(function(err, tags) {
-                      expect(tags.length).to.equal(1);
-                      callback();
-                    });
-                  }
-                ], done);
+          return this.sequelize.sync().then(function() {
+            return Promise.all([
+              self.Product.bulkCreate([
+                {title: 'Chair'},
+                {title: 'Desk'},
+                {title: 'Handbag'},
+                {title: 'Dress'},
+                {title: 'Jan'}
+              ]),
+              self.Tag.bulkCreate([
+                {name: 'Furniture'},
+                {name: 'Clothing'},
+                {name: 'People'}
+              ])
+            ]).then(function() {
+              return Promise.all([
```

Comment:
I don't really care, eventhough I prefere to use all, so it's a bit less to read ;).
Shall I change all entries to use props?